kubetest/e2e.go

```diff
@@ -506,16 +518,21 @@ func KubemarkTest(dump string, o options) error {
 	})
 
 	// Start new run
-	if popN, err := pushEnv("NUM_NODES", o.kubemarkNodes); err != nil {
+	if popN, err := pushEnv("NUM_NODES", nodes); err != nil {
 		return err
 	} else {
 		defer popN()
 	}
-	if popM, err := pushEnv("MASTER_SIZE", o.kubemarkMasterSize); err != nil {
+	if popM, err := pushEnv("MASTER_SIZE", masterSize); err != nil {
```

Comment:
I think we want as many parameters as possible to be calculated automatically, to make our setup less error-prone. We are doing this for a lot of variables like master-size, master-disk-size, master-root-disk-size, etc in the case of normal clusters. IMO we can get rid of this in kubemark too and I checked that in fact all master sizes for kubemark jobs being passed currently are same as what would've been auto-calculated. You just need to remove all code around the kubemarkMasterSize flag/envvar and nothing should break :)